### PR TITLE
[WIP] retroarch: unmark broken on arm*

### PR DESCRIPTION
**Pull request by ndowens**
Signed-off-by: Nathan Owens <ndowens04@gmail.com>

### CODE REVIEW

File: srcpkgs/retroarch/template [hunk 1]
```diff
@@ -29,12 +29,11 @@ build_options="ffmpeg opengl jack pulseaudio sdl2 x11 vulkan qt5"
 build_options_default="ffmpeg"
 
 case "$XBPS_TARGET_MACHINE" in
-	i686*|x86_64*|ppc64*) build_options_default+=" opengl pulseaudio sdl2 x11 vulkan";;
-	armv[67]*) broken="https://build.voidlinux.org/builders/armv7l_builder/builds/19214/steps/shell_3/logs/stdio"
+	i686*|x86_64*|ppc64*|arm[67]) build_options_default+=" opengl pulseaudio sdl2 x11 vulkan";;
```
Comment (Hoshpak) [inline srcpkgs/retroarch/template]: what's the reason for the change in this line? We don't have architectures that are called armv6 or armv7.

Comment (ndowens) [reply srcpkgs/retroarch/template]: Will fix
